Kimjin/3주차 과제/과제1/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Kim Jin</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="box1">
```

Comment:
클래스 이름은 의미를 파악할 수 있도록 지어주는 게 좋아요. 그래야만 합니다!
그래서 `box1`, `box2`보다 의미가 있는 이름으로 지어주면 좋을 것 같아요.
예를 들면 `box1`은 `explain-box`, `box2`는 `contents-box` 와 같이요!
```suggestion
    <div class="explain-box">
```